libraries/AP_InertialSensor/AP_InertialSensor.cpp

```diff
@@ -861,14 +861,6 @@ AP_InertialSensor::init(uint16_t loop_rate)
         _start_backends();
     }
 
-    // initialise accel scale if need be. This is needed as we can't
-    // give non-zero default values for vectors in AP_Param
-    for (uint8_t i=0; i<get_accel_count(); i++) {
-        if (_accel_scale[i].get().is_zero()) {
```

Comment:
as above we do now have the possibility to do the wrong thing if all zeros are saved in eeprom.